samples/javascript_nodejs/11.qnamaker/package.json

```diff
@@ -16,9 +16,10 @@
         "url": "https://github.com/Microsoft/BotBuilder-Samples.git"
     },
     "dependencies": {
-        "botbuilder": "^4.3.0",
-        "botbuilder-ai": "^4.3.0",
-        "botframework-config": "^4.3.0",
+        "botbuilder": "~4.3.4",
+        "botbuilder-ai": "~4.3.4",
+        "botframework-config": "~4.3.4",
+        "botframework-schema": "^4.3.4",
```

Comment:
So we want to get rid of the `botframework-config` library because it shouldn't be used at runtime. You also shouldn't need the `botframework-schema` because it just provides TS interfaces, which aren't consumed at runtime.

If you still need the `botframework-schema` package it's exposed through `botbuilder`.

Reply:
Done. Just need to lint and test.